queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] This query suppose to find all animals that its name ends with `mon`, but you did make it find all animals that contain `mon` in its name. Kindly fix that.
[Hint] It will be like that instead:
```sql
SELECT * FROM animals WHERE name LIKE '%mon';
```